src/lib/deploy.js

```diff
@@ -500,6 +525,13 @@ function getAccountQuery(publicKey) {
   }`;
 }
 
+function getStatusQuery() {
+  return `
+  query {
```

Comment:
Not sure it's worth separating this out. Would it make sense to put this in Line 163?